.github/workflows/sync-branches.yaml

```
@@ -0,0 +1,40 @@
+name: load-rs::sync branches
+
+# The goal of this workflow is to create PR with changes of load-rs script itself
+#   from iroha2-dev branch because we need to keep the same rust source code for all script versions
+# Ignore changes to Cargo.toml since this file should be updated from load-rs::bump-dependencies workflow
+# No need to touch iroha2-lts branch because it is updated once per six months and should be stable
+
+on:
+ pull_request:
+  branches: [iroha2-dev]
+  types: [closed]
+
+jobs:
+  merge_iroha2-dev_to_master:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 1
+      - name: Filter files to be merged
+        uses: dorny/paths-filter@v2.10.2
+        id: filter
+        with:
+          base: master
+          filters: |
+            cargo:
+              - 'Cargo.toml'
+      - name: Create pull request
+        if: steps.filter.outputs.cargo == 'false'
+        uses: vsoch/pull-request-action@1.0.19
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PULL_REQUEST_BRANCH: "master"
```

This is a chance to make a modern change:

```suggestion
          PULL_REQUEST_BRANCH: "main"
```

---

That is completely unnecessary, given that our main branch is called `iroha2`.